SuperglobalsAndFormHandling/superGlobal.php

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Simple Form</title>
+</head>
+<body>
+    <h2>Simple Form</h2>
+
+    <?php
+    if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+        $name = $_POST['name'];
+        $email = $_POST['email'];
+
+        if (!empty($name) && !empty($email)) {
+            echo "Name: " . htmlspecialchars($name) . "<br>";
+            echo "Email: " . htmlspecialchars($email) . "<br>";
+        } else {
+            echo "Please fill in both fields.";
+        }
+    } else {
```

Comment:
There is nothing wrong with this but use colon for better readability.

Instead of using {}, you use a colon : to start the block and endif